knative-operator/Gopkg.toml

```diff
@@ -51,6 +51,10 @@ required = [
 #   name = "knative.dev/pkg"
 #   branch = "release-0.14"
 
+[[override]]
+  name = "knative.dev/serving"
+  branch = "release-0.14"
```

Comment:
should probably be 0.16 now.

Reply:
I opted for `0.15` to be on pair with ingress operator and also `knative.dev/operator`.